ko/spec/ics-001-ics-standard/README.md

```diff
@@ -0,0 +1,152 @@
+---
+ics: '1'
+title: ICS 사양 표준
+stage: 초안
+category: meta
+author: 'Christopher Goes '
+created: '2019-02-12'
+modified: '2019-08-25'
+---
+
+## ICS 란 무엇인가?
+
+인터체인 표준(Inter-chain standard, ICS)은 특정 프로토콜, 표준, 또는 코스모스 생태계가 사용될 것이라 기대되는 특징들을 서술하는 설계 문서입니다. ICS는 표준에서 필요한 특성들을 나열하고, 설계 근거를 설명하며, 간결하고 포괄적인 기술적인 사양들을 제공해야 합니다. ICS 저자는 표준화 프로세스를 통해 제안을 추진하고, 커뮤니티의 지원을 요청하며, 사회적 합의를 위해 관련 이해 관계자와 의사소통을 해야합니다.
+
+인터 체인 표준화 프로세스는 생태계 전반의 프로토콜들과 변화, 특징들을 제안하는 주요 수단이 되어야 하며, ICS 문서들은 향후 구현자들을  위해 설계 결정사항과 각종 정보들을 합의 후에도 유지해야 합니다.
+
+인터체인 표준은 특정 블록체인(코스모스 허브와 같은)의 변화를 제안하거나, 특정 구현 특징들(언어 별 데이터 구조 등), 또는 코스모스 블록체인들 위에서 발생하는 사용되어서는 안됩니다.(비록, 코스모스 생태계에 있는 개별 블록체인들은 인터체인 표준을 사용할지 말지를 정할 거버넌스를 이용할 수 있지만)
+
+## 구성요소
+
+ICS는 헤더, 개요, 명세, 히스토리 로그, 저작권 공지사항으로 구성됩니다. 레퍼런스는 반드시 링크 또는 필요시 하단의 표를 포함해야 합니다.
+
+### 헤더
+
+ICS 헤더는 관련된 메타데이터를 포함합니다.
+
+#### 필수 필드
+
+`ics: #` - ICS 번호 (순차적으로 할당됩니다.)
+
+`title` - ICS 제목 (짧고 명료하게 표현해야 합니다.)
+
+`stage` - 현재 ICS 단계를 나타내며, 사용할 수 있는 단계들은 [PROCESS.md](../../PROCESS.md) 에서 확인할 수 있습니다.
+
+ICS에서 사용가능한 단계들에 대한 설명은 [README.md](../../README.md)을 참조하십시오.
+
+`category` - ICS 카테고리, 다음 중 하나입니다:
+
+- `meta` - ICS 단계와 관련된 표준
+- `IBC/TAO` - 블록체인간 통신에서의 전송, 인증, 순서 정렬과 같은 코어 시스템 표준
+- `IBC/APP` - 블록체인간 통신 시스템 어플리케이션 레이어 프로토콜 표준
+
+`author` - ICS 저자 & 계약 정보 (선호 순서 : 이메일, Github, Twitter, 기타 응답을 요구할 수 있는 연락 방법).
+제 1 저자는 기본적으로 ICS의 "소유자(owner)"이고, 표준화 과정을 통해 발전시켜야 할 책임이 있습니다. 이어지는 저자들의 순서는 기여한 양에 따라 정해져야 합니다.
+
+`created` - ICS가 처음 만들어진 날짜 (`YYYY-MM-DD`)
+
+`modified` - ICS가 마지막으로 수정된 날짜 (`YYYY-MM-DD`)
+
+#### 선택적 필드
+
+`requires` - 이 표준에서 요구하거나 의존하는 다른 ICS 표준(번호에 의해 참조됩니다.)
+
+`required-by` - 이 표준을 요구하거나 의존하는 다른 ICS 표준 (번호에 의해 참조됩니다.)
+
+`replaces` - 해당되는 경우, 다른 ICS 표준으로 대체되거나, 이 표준으로 대체되는 표준
+
+`replaced-by` - 해당되는 경우, 다른 ICS를 대체하거나 이 표춘을 대체하는 표준
+
+### 개요 (Synopsis)
+
+헤더에 이어지며, ICS는 반드시 고수준의 설명과 관련된 명세를 제공하는 200자 이내의 간략한 개요를 포함해야 합니다.
+
+### 명세 (Specification)
+
+명세는 ICS의 메인 구성요소이고, 프로토콜 문서, 설계의 이론적 해석, 요구되는 레퍼런스, 그리고 적절한 기술적인 내용들을 포함해야 합니다.
+
+#### 하위 구성요소 (Sub-components)
+
+명세는 특정 ICS에 적합한 여러 종류의 하위 구성요소를 가질 수 있습니다. 포함되는 하위 구성요소는 아래에 설명된 순서대로 나열돼야 합니다.
+
+- *Motivation* - 제안된 특징의 이론적인 해석, 또는 기존 특징을 바꾸는 제안
+- *Definitions* - 이 ICS의 새로운 용어나 컨셉, 또는 이 ICS를 이해하기 위해 요구되는 목록. 가장 상위 docs 폴더에서 정의되지 않은 용어들은 반드시 여기서 정의되어야 합니다.
+- *Desired Properties* - 프로토콜 또는 명시된 기능에서 원하는 속성, 그리고 기대 효과 또는 속성들이 위반되었을 경우의 실패 목록입니다.
+- *Technical Specification* - 문맥과 시멘틱, 하위 프로토콜, 자료구조, 알고리즘과 적절한 의사코드를 포함하는 모든 기술적인 내용입니다. 기술 명세는 상호간의 지식이 없는 분리된 구현체들이 호환될 수 있도록 충분히 상세해야 합니다.
+- *Backwards Compatibility* - 이전의 특징 또는 프로토콜 버전과의 호환성(또는 부족)에 대한 설명
```

Comment:
```suggestion
- *Backwards Compatibility* - 이전 기능 또는 프로토콜 버전과의 호환성(또는 이에 대한 부족) 논의
```

Reply:
I think "lack thereof" means, deprecated feature compared to previous version.